extensions/positron-python/python_files/posit/test-requirements.txt

```diff
@@ -9,11 +10,10 @@ matplotlib
 numpy
 pandas
 plotly
-polars; python_version >= '3.9'
+polars
 polars[timezone]; sys_platform == 'win32'
-pyarrow; python_version < '3.13'
-pyarrow>=19.0.0; python_version >= '3.13'
-pytest<8.1.1
+pyarrow
+pytest
```

Comment:
These two issues seem to be fixed. We no longer need an old pytest or different versions of Arrow for different versions of Python.